src/foxops/engine/rendering.py

```diff
@@ -14,6 +15,20 @@
 logger = get_logger(__name__)
 
 
+def first_ip_address_is_greater(first_ip, second_ip):
```

Comment:
I think we should move these filters into a separate file, i.e. `custom_filters.py` and also add some separate unit tests of these functions directly.